Broken travis.ci tests.

All the travis.ci builds (Go 1.6 - 1.8) fail with the following error:

```
>> vetting code
# honnef.co/go/tools/lint/lintutil
../../../honnef.co/go/tools/lint/lintutil/util.go:273: undefined: types.SizesFor
make: *** [/home/travis/gopath/bin/staticcheck] Error 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,9 +3,6 @@ sudo: false
 language: go
 
 go:
-- "1.6.x"
-- "1.7.x"
-- "1.8.x"
 - "1.9.x"
 - "1.10.x"
 

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # Prometheus Mesos Exporter
 
-[![Build Status](https://travis-ci.org/mesosphere/mesos_exporter.svg?branch=master)](https://travis-ci.org/mesosphere/mesos_exporter)
+[![Build Status](https://travis-ci.org/mesos/mesos_exporter.svg?branch=master)](https://travis-ci.org/mesos/mesos_exporter)
 
 Exporter for Mesos master and agent metrics.
 

